src/Microsoft.Kiota.Cli.Commons.Tests/IO/JsonOutputFormatterTest.cs

```diff
@@ -1,74 +1,75 @@
-﻿using System.IO;
+﻿using System;
+using System.IO;
 using System.Text;
 using System.Threading.Tasks;
 using Microsoft.Kiota.Cli.Commons.IO;
-using Spectre.Console.Testing;
+using Microsoft.Kiota.Cli.Commons.Tests.Fakes;
 using Xunit;
 
 namespace Microsoft.Kiota.Cli.Commons.Tests.IO;
 
-public class JsonOutputFormatterTest
+public sealed class JsonOutputFormatterTest
 {
     public class WriteOutputAsyncFunction_Should
     {
-        private readonly TestConsole _console;
-
-        private const string _newLine = "\n";
-
-        public WriteOutputAsyncFunction_Should()
-        {
-            _console = new TestConsole();
-        }
+        private readonly string NewLine = Environment.NewLine;
 
         [Fact]
         public async Task Write_A_Line_With_Stream_Content()
         {
-            var formatter = new JsonOutputFormatter(_console);
+            var tc = new TestConsole();
+            var formatter = new JsonOutputFormatter(tc);
             var content = "Test content";
             var stream = new MemoryStream(Encoding.ASCII.GetBytes(content));
 
             await formatter.WriteOutputAsync(stream, new JsonOutputFormatterOptions(true));
 
-            Assert.Equal($"{content}{_newLine}", _console.Output);
+            Assert.Equal($"{content}{NewLine}", tc.Output);
         }
 
         [Fact]
         public async Task Write_Indented_Output_Given_A_Minified_Json_Stream()
         {
-            var formatter = new JsonOutputFormatter(_console);
+            var tc = new TestConsole();
+            var formatter = new JsonOutputFormatter(tc);
             var content = "{\"a\": 1, \"b\": \"test\"}";
             var stream = new MemoryStream(Encoding.ASCII.GetBytes(content));
-            var n = _newLine;
+            var n = NewLine;
 
             await formatter.WriteOutputAsync(stream, new JsonOutputFormatterOptions(true));
             var expected = $"{{{n}  \"a\": 1,{n}  \"b\": \"test\"{n}}}";
+            Console.Out.Flush();
```

Comment:
This looks to be unnecessary. Is it needed?
Otherwise, maybe use `await Console.Out.FlushAsync();`
Same for the other modified tests

Reply:
Good catch. This was left over from an earlier change. Thanks.